integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -2,6 +2,7 @@
 from datetime import datetime, timedelta
 from typing import List, Tuple, Any, Union, TYPE_CHECKING
 
+import anyio.to_thread
```

Comment:
can u import this more specifically?

Reply:
talked offline, decide to leave it like it is for better readability, as it is not much used function so its better to have the full namespacing when using the function rather than just the function name which makes the code less readable and requires hoping to the import line to find what package it is related to